gke/node_service_account/main.tf

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
This file duplicates all the contents of autopilot/custom_service_account, and may not be required. If it is required, it must contain unique region tags.